src/NBomber/Infra/Http.fs

```diff
@@ -24,60 +24,60 @@ let serializeBinary (data: obj) =
     let formatter = BinaryFormatter()
     formatter.Serialize(ms, data)
     ms.ToArray()
-    
-let deserializeBinary<'T>(data: byte[]) = 
+
+let deserializeBinary<'T>(data: byte[]) =
     use ms = new MemoryStream(data)
     let formatter = BinaryFormatter()
-    formatter.Deserialize(ms) :?> 'T 
+    formatter.Deserialize ms :?> 'T

```

Comment:
why remove here (), it's actually not clear now the invocation flow, I would then rewrite
ms |?> T |> formatter.Seserilize